refactor(edit-content): enhance tab panel structure and iframe styling

oidacra opened the pull request:
### Proposed Changes

- Updated the first tab panel to follow a consistent structure with other tabs by wrapping the content in an ng-template.
- Modified the custom field component to include a minimum height for the container, improving layout consistency.
- Enhanced iframe handling by adding styles to hide scrollbars and prevent continuous height growth, ensuring a smoother user experience.
- Added new computed property for minimum container height based on label visibility, improving responsiveness.

### Checklist
- [ ] Tests
- [ ] Translations
- [ ] Security Implications Contemplated (add notes if applicable)

### Additional Info
** any additional useful context or info **

### Screenshots
Original             |  Updated
:-------------------------:|:-------------------------:
** original screenshot **  |  ** updated screenshot **

This PR fixes: #31763

oidacra committed:
refactor(edit-content): enhance tab panel structure and iframe styling

- Updated the first tab panel to follow a consistent structure with other tabs by wrapping the content in an ng-template.
- Modified the custom field component to include a minimum height for the container, improving layout consistency.
- Enhanced iframe handling by adding styles to hide scrollbars and prevent continuous height growth, ensuring a smoother user experience.
- Added new computed property for minimum container height based on label visibility, improving responsiveness.

oidacra committed:
refactor(edit-content): remove unused JsonPipe from custom field component imports

- Eliminated the  import from  as it was not being utilized, streamlining the component's dependencies.

oidacra committed:
refactor(edit-content): remove component

oidacra committed:
fix(edit-content): fix tests

nicobytes commented on core-web/libs/edit-content-bridge/src/lib/factories/form-bridge.factory.ts:
```diff
@@ -5,18 +5,35 @@ import { AngularFormBridge } from '../bridges/angular-form-bridge';
 import { DojoFormBridge } from '../bridges/dojo-form-bridge';
 import { FormBridge } from '../interfaces/form-bridge.interface';
 
+/**
```
Great! The JSDocs comments are missing in this part

nicobytes commented on core-web/libs/edit-content/src/lib/components/dot-edit-content-form/dot-edit-content-form.component.html:
```diff
@@ -32,11 +35,13 @@
 
-            <!-- First tab is not lazy -->
+            <!-- First tab with same pattern as other tabs -->
             @if ($firstTab()) {
```
I suggest creating a let variable for `$firstTab()` because it is used more times below.

oidacra replied:
I did something more deeper here.

nicobytes commented on core-web/libs/edit-content/src/lib/fields/dot-edit-content-custom-field/dot-edit-content-custom-field.component.html:
```diff
@@ -1,15 +1,19 @@
+            (load)="onIframeLoad()"
+            [src]="src | safeUrl"
+            [class.legacy-custom-field--fullscreen]="$isFullscreen()"
+            [ngStyle]="
```
The `ngStyle` directive will be deprecated in the future, please use the `[style]` directive instead

oidacra replied:
done.

oidacra committed:
refactor(edit-content): streamline tab handling and improve iframe styles

- Replaced the logic for retrieving tabs in the edit content form component, consolidating it into a single computed property for better readability and maintainability.
- Updated the tab panel structure to enhance consistency by removing redundant code.
- Improved iframe styling in the custom field component to ensure proper height and width handling, enhancing the user experience.
